### PR TITLE
Task2. S3, Cloudfront

### PR DESCRIPTION
**Pull request by MaxGaranin**
Manual configured S3/CloudFrond resources:  
S3 link (403) - http://rss-aws-app.s3-website-eu-west-1.amazonaws.com/  
CloudFront link - https://d3rdi212t1dvge.cloudfront.net/  

Automate configured S3/CloudFrond resources:   
S3 link (403) - http://rss-second-aws-app.s3-website-eu-west-1.amazonaws.com/
CloudFront link - https://d3gqyf3iecrlqi.cloudfront.net

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,14 @@
+## Links for Task2  
+
+Manual configured S3/CloudFrond resources:  
+S3 link - http://rss-aws-app.s3-website-eu-west-1.amazonaws.com/  
+CloudFront link - https://d3rdi212t1dvge.cloudfront.net/  
+
+Automate configured S3/CloudFrond resources:  
+S3 link - http://rss-second-aws-app.s3-website-eu-west-1.amazonaws.com/  
+CloudFront link - https://d3gqyf3iecrlqi.cloudfront.net
```
Comment (aosin) [inline README.md:9]: This link doesn't work. I'm getting an error:
```xml
<Error>
<Code>AuthorizationHeaderMalformed</Code>
<Message>The authorization header is malformed; the region 'us-east-1' is wrong; expecting 'eu-west-1'</Message>
<Region>eu-west-1</Region>
<RequestId>33067E35BFF5260B</RequestId>
<HostId>gOakwBhPt13IEEr7Ghrgs/XTmzaU9LO+VKP5G6AQ44kAXLkuevGKRG//0EqT0uIWdT2h1iA7Caw=</HostId>
</Error>
```

File: package.json [hunk 2]
```diff
@@ -5,34 +5,49 @@
   "scripts": {
     "start": "react-scripts start",
     "build": "react-scripts build",
+    "deploy:s3": "serverless client deploy --no-config-change --no-policy-change --no-cors-change",
+    "build:deploy": "npm run build && npm run deploy:s3",
```
Comment (aosin) [inline package.json]: You probably forgot to remove this after adding "client:deploy" and "client:build:deploy"

File: .gitignore [hunk 3]
```diff
@@ -25,3 +25,9 @@ yarn-error.log*
 
 /.idea
 .vscode
+
+# package directories
+jspm_packages
+
+# Serverless directories
+.serverless
```
Comment (aosin) [inline .gitignore]: It is customary to end each source file with a new line. Helps with cleaner diffs when adding stuff to the bottom, among other things.

File: README.md (hunk 1 above)
Comment (aosin) [reply README.md:9]: You probably moved your bucket to eu-west-1 after having initially deployed it to us-east-1. Try re-deploying the CF stack. You may want to update this link afterwards.

**Review by aosin — CHANGES_REQUESTED** [review]
Please fix the broken link to cloudfront distribution

File: README.md (hunk 1 above)
Comment (MaxGaranin) [reply README.md:9]: Eventually, a couple a days later, link was recovered by Amason. Now it's working.

File: package.json (hunk 2 above)
Comment (MaxGaranin) [reply package.json]: Fixed.

File: .gitignore (hunk 3 above)
Comment (MaxGaranin) [reply .gitignore]: Fixed.

**Review by aosin — APPROVED** [review]

**Pull request merged**